music21/stream/base.py

```diff
@@ -5371,14 +5371,13 @@ def toSoundingPitch(self, *, inPlace=False):
         'C3'
 
         If 'atSoundingPitch' is unknown for this Stream and all of its parent Streams
-        then will raise a StreamException:
+        then no transposition will take place (this used to raise an exception):
 
         >>> s = stream.Score()
         >>> p = stream.Part(id='partEmpty')
         >>> s.append(p)
```

Comment:
Oh, while we're here, can we change this to `s.insert(0.0, p)` -- trying to get people not to get in the habit of appending Part objects to Scores (makes no difference here, but just good form)

Reply:
done.